.github/workflows/release.yml

```diff
@@ -14,6 +14,17 @@ jobs:
     runs-on: ubuntu-latest
     needs:
       - build
+    env:
+      dependencies: |
+        docutils>=0,<1
+        guzzle-sphinx-theme>=0,<1
+        m2r2>=0,<1
+        pytest-cov>=2,<5
+        sphinx>=4,<6
```

Comment:
see the pending pull request that got opened by dependabot. They keep the range >=3, <6 for sphinx for some reason. Probably an idea to follow?